documentation/raw_data_import_process.md

```diff
@@ -0,0 +1,84 @@
+# Raw Data Import Process
+
+## Purpose
+The stakeholder will need to import a raw dataset into the database once a year. 
+A script is useful to help automate this task.
+The stakeholder is able to use a command line to initiate the script.
+
+## Overview
+- Use a Python script to clean the original `Asylum Case Data` and convert to .csv 
+- Use AWS implementation for bulk .csv ingestion to import cleaned data to DynamoDB
+
+## Import Process
+### Prerequisites
+1. IAM User with AWS permissions
+   - CloudFormation
+   - DynamoDB 
+   - Lambda
+   - S3 
+2. AWS CLI Installed _(Latest Version)_
+      - aws_access_key_id
+      - aws_secret_access_key
+3. Python3 Installed with Pandas Library _(Latest Versions)_
+
+### Requirements
+- [COW2021001887-I589Data.xlsx](https://bloomtech.notion.site/HRF-Asylum-Report-Generator-412062ca7640457682c33295b21a25c3#:~:text=FE%3A%20React%20(JS)-,Resources,-Asylum%20decision%20data) _(Resources Section)_
+- [AsylumCaseXlsxToCleanCsv.py](/configurations/AsylumCaseXlsxToCleanCsv.py)
+- [CSVToDynamoDB.template.json](/configurations/CSVToDynamo.template.json)
+
+### Importing
+#### Deploy Stack 
+1. Open CLI in the root of this project
+2. CLI command for us-west-2 `aws cloudformation create-stack --region us-west-2 --stack-name asylum-rg-be-a --template-body file://configurations/CSVToDynamo.template.json --capabilities CAPABILITY_IAM`
+3. Stack will build DynamoDB Table `asylum_cases`, S3 Bucket named `asylum-cases` with lambda trigger, and Lambda Function
+
+#### Clean Data
+The current configuration requires the [Excel file](https://bloomtech.notion.site/HRF-Asylum-Report-Generator-412062ca7640457682c33295b21a25c3#:~:text=FE%3A%20React%20(JS)-,Resources,-Asylum%20decision%20data) 
+and [Python script](/configurations/AsylumCaseXlsxToCleanCsv.py) to reside in the same directory
+1. Place files in the same directory `(ex. Data_Import/_both files_)`
+2. Change the name of the Excel file to `data.xlsx`
+3. Open CLI in the same path as the files
+4. Command `python3 AsylumCaseXlsxToCleanCsv.py`
+5. This will generate `asylum-cases-data.csv` in the same directory in accordance with S3 bucket naming standards<sup>2</sup>
+  
+#### Upload Data to S3 Bucket
+- Open CLI in the same directory as `asylum-cases-data.csv`
+- CLI command `aws s3 cp <your-directory>/asylum-cases-data.csv s3://asylum-cases/asylum-cases-data.csv`
+
+## Functionality
+### Cleaning Data
+The [script](/configurations/AsylumCaseXlsxToCleanCsv.py) reads the .xlsx file into Pandas<sup>5, 6</sup> DataFrame. It then generates a UUID for each row and 
```

Comment:
I think this will be a broken link because the file name in the repo is:
asylumCaseXlsxToCleanCsv.py